packages/react-core/src/components/Page/examples/GroupSection.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Page,
+  Masthead,
+  MastheadToggle,
+  MastheadMain,
+  MastheadBrand,
+  MastheadContent,
+  PageSidebar,
+  PageSection,
+  PageGroup,
+  PageBreadcrumb,
+  PageNavigation,
+  PageSectionVariants,
+  PageToggleButton,
+  Breadcrumb,
+  BreadcrumbItem,
+  Nav,
+  NavList,
+  NavItem,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem
+} from '@patternfly/react-core';
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+
+export const GroupPage: React.FunctionComponent = () => {
+  const [isNavOpen, setIsNavOpen] = React.useState(true);
+
+  const onNavToggle = () => {
+    setIsNavOpen(!isNavOpen);
+  };
+
+  const headerToolbar = (
+    <Toolbar id="group-section-toolbar">
+      <ToolbarContent>
+        <ToolbarItem>header-tools</ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  const Header = (
```

Comment:
I think that the convention is to use camelCase, is that correct @nicolethoen?

```suggestion
  const header = (
```

Reply:
That's correct - we generally use camelCase for our variable names and PascalCase for component names. Then any text, whether prop description or example titles are all Sentence case.